src/quantum/azext_quantum/_help.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['quantum'] = """
+    type: group
+    short-summary: Manage Azure Quantum Workspaces and submit jobs to Azure Quantum Providers.
```

Comment:
pls provide example for each commands, exmaple: https://github.com/Azure/azure-cli-extensions/blob/master/src/blueprint/azext_blueprint/_help.py#L20